server/aws-lsp-codewhisperer/src/language-server/telemetryService.ts

```diff
@@ -119,6 +120,30 @@ export class TelemetryService extends CodeWhispererServiceToken {
             numberOfRecommendations: session.suggestions.length,
             perceivedLatencyMilliseconds: perceivedLatencyMilliseconds,
         }
-        this.invokeSendTelemetryEvent(event)
+
+        this.invokeSendTelemetryEvent({
+            userTriggerDecisionEvent: event,
+        })
+    }
+
+    public emitCodeCoverageEvent(params: {
```

Comment:
This function will be used to emit CodeCoverage events to toolkits DWH as well. Can we change the function arguments so it has the superset fields to be used for both STE and Toolkits DWH?

Reply:
Let's do this as part of follow-up refactoring task. This params based on CodeCoverage event